pyproject.toml

```diff
@@ -70,6 +70,8 @@ validators = "^0.34.0"
 django-flags = "^5.0.13"
 insights-analytics-collector = "^0.3.2"
 distro = "^1.9.0"
+dispatcherd = { version = "v2025.05.12", extras = ["pg_notify"] }
```

Comment:
@Alex-Izquierdo please update to 05.19 as I built that version instead of 05.12 due to license issue.

Reply:
@simaishi Done!